osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -0,0 +1,169 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package packagemanagement
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	osconfigpb "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/_internal/gapi-cloud-osconfig-go/google.golang.org/genproto/googleapis/cloud/osconfig/v1alpha1"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/compute"
+	osconfigserver "github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/osconfig_server"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/utils"
+	api "google.golang.org/api/compute/v1"
+)
+
+var (
+	pkgTestSetup = []*packageManagementTestSetup{}
+	pkgManagers  = [...]string{"apt"}
+)
+
+var vf = func(inst *compute.Instance, vfString string, port int64, interval, timeout time.Duration) error {
+	if err := inst.WaitForSerialOutput(vfString, port, interval, timeout); err != nil {
+		return err
+	}
+	return nil
+}
+
+func addCreateOsConfigTest() {
+	testName := "createosconfigtest"
+	desc := "test osconfig creation"
+	for _, pkgManager := range pkgManagers {
+		var oc *osconfigpb.OsConfig
+
+		switch pkgManager {
+		case "apt":
+			pkg := osconfigserver.BuildPackage("cowsay")
+			pkgs := []*osconfigpb.Package{pkg}
+			oc = osconfigserver.BuildOsConfig(testName, desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
+		}
```

Comment:
Can you add a `default` case to catch any future mistakes as we add support for additional pkg manager types? It can just log or panic.

Reply:
yeah sure.. done

Reply:
Don't panic, if you need error handling add error handling, or just print an error as this really may not matter much.